mgmtsystem_action/models/mgmtsystem_action.py

```diff
@@ -19,11 +33,14 @@ def _default_owner(self):
         """Return the user."""
         return self.env.user
 
+    @api.model
     def _default_stage(self):
         """Return the default stage."""
-        return self.env['mgmtsystem.action.stage'].search(
-            [('is_starting', '=', True)],
-            limit=1)
+        return (
+            self.env.ref('mgmtsystem_action.stage_draft', False) or
```

Comment:
@grindtildeath 
The prior version of the method returned the xmlid of the stage, stage_draft, meaning the stage with id stage_draft had to be present at all times in the database for the create method to work, now if a manager user deletes the stage with the id stage_draft it can no longer be retrieved by the method and create method for new records fails. So this new method checks for there being a stage with id stage_draft and if this stage is not present it returns the stage that has the bool field as being the starting stage.